lib/CPathfinder.h

```diff
@@ -54,20 +69,61 @@ struct DLL_LINKAGE CGPathNode
 		BLOCKED //tile can't be entered nor visited
 	};
 
+	STRONG_INLINE
+	void setCost(float value)
+	{
+		if (value == cost._cost) {
+			return;
+		}
+		bool getUpNode = value < cost._cost;
+		cost._cost = value;
+		// If the node is in the heap, update heap.
+		if (inPQ && pq != nullptr)
```

Comment:
No space after `if` please,
```suggestion
		if(inPQ && pq != nullptr)
```